uvicgse/project-virtual-team-1
# Issue 95 - Add Remote Upstream Repository

This implements issue #95 
To test:
1. Click "remote" and enter the url of the remote upstream repository and confirm
2. cd into your project repo and type `git remote -v`
3. The upstream repository should appear in the list
